tests/servers/tests.py

```diff
@@ -174,3 +175,23 @@ def test_database_writes(self):
             ['jane', 'robert', 'emily'],
             lambda b: b.name
         )
+
+
+class LiveServerPort(LiveServerBase):
+
+    def test_port_bind(self):
+        """
+        Ensure that each LiveServerTestCase binds to a unique port
+        or fails to start a server thread when run concurrently.
+        Refs #26011.
+        """
+        TestCase = type(str("TestCase"), (LiveServerBase,), {})
+        try:
+            TestCase.setUpClass()
+        except socket.error as e:
+            self.assertEqual(e.errno, errno.EADDRINUSE)
```

Comment:
could you add comments to explain the test a bit more? e.g. under what circumstances do we hit this error as opposed to continuing to the rest of the test.